years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
```

Comment:
@mkordas please, look at the [build log](https://travis-ci.org/yegor256/rultor/builds/112354939)  and see building javadocs before years.sh `mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V.` This goal makes `target` folder. 
Firstly I didn't include target folder and got violation ([see failed travis log](https://travis-ci.org/yegor256/rultor/builds/112348557))